# HG changeset patch
# User andrew
# Date 1532526839 -3600
#      Wed Jul 25 14:53:59 2018 +0100
# Node ID 66087a86ded39c2595dc61223fa03671ff051e33
# Parent  2f2d2af6fa5c44e67e0a9987f56392315a1e4b64
8207057, PR3613: No debug info for assembler files

diff --git openjdk.orig/hotspot/make/linux/makefiles/gcc.make openjdk/hotspot/make/linux/makefiles/gcc.make
--- openjdk.orig/hotspot/make/linux/makefiles/gcc.make
+++ openjdk/hotspot/make/linux/makefiles/gcc.make
@@ -334,6 +334,7 @@
 # DEBUG_BINARIES uses full -g debug information for all configs
 ifeq ($(DEBUG_BINARIES), true)
   CFLAGS += -g
+  ASFLAGS += -g
 else
   # Use the stabs format for debugging information (this is the default
   # on gcc-2.91). It's good enough, has all the information about line
